### DISCUSSION TITLE
Enhancement: Auto hotlink images to a full-size view

### DISCUSSION THREAD

**Discussion by kobaz** [requarks/wiki#5661] [General]
When adding an image to a page and resizing it down, there's no clickable or linked item to be able to go to the full-size image.

It would be great to have a pop-out <div> what has pan/zoom options when an image has been added to a page

**Comment by NGPixel** [comment]
Stop posting enhancement requests here. This site is for questions / help only.

All your requests are already posted here:
https://requarks.canny.io/wiki
